internal/command/views/show.go

```diff
@@ -58,7 +60,10 @@ type ShowHuman struct {
 
 var _ Show = (*ShowHuman)(nil)
 
-func (v *ShowHuman) DisplayState(stateFile *statefile.File, schemas *tofu.Schemas) int {
+func (v *ShowHuman) DisplayState(ctx context.Context, stateFile *statefile.File, schemas *tofu.Schemas) int {
+	_, span := tracing.Tracer().Start(ctx, "Display")
+	defer span.End()
```

Comment:
Having view-level code generate traces feels a little weird to me, since they should never be doing anything other than preparing a buffer to write to the output streams and then writing to those streams, and so I'd personally consider this to be noise rather than signal. :thinking: 

Are you doing this with a concern that there might be some view implementations that are actually doing more than they ought to? I could understand that instinct, but that seems like it's more in the realm of an implementation bug that we should fix rather than something that an end-user should be worrying about, and so doesn't seem in scope for our current goals.

Reply:
I think I've found out via experience and testing what you've figured out from intuition here!  

I realise that this `Display` span is very useless and I really think that we can drop this. 

My initial concern was if json serialization takes too long in `-json` however I think this is negligable compared to the rest of the span and I think we don't need an explicit span for this.